src/integrations/ga/GaToSplit.js

```diff
@@ -1,7 +1,257 @@
+
+  // Filter based on rum-agent identities validator
+  return uniqueIdentities.filter(identity => {
+    if (!identity)
+      return false;
+
+    const maybeKey = identity.key;
+    const maybeTT = identity.trafficType;
+
+    if (!isString(maybeKey) && !isNumber(maybeKey))
+      return false;
+    if (!isString(maybeTT))
+      return false;
+
+    return true;
+  });
+}
+
+/**
+ * Validates if a given object is a EventData instance, and logs corresponding warnings.
+ *
+ * @param {EventData} data event data instance to validate. Precondition: data != undefined
+ * @returns {boolean} Whether the data instance is a valid EventData or not.
+ */
+export function validateEventData(data) {
+  if (!validateEvent(data.eventTypeId, 'splitio-ga-to-split:mapper'))
+    return false;
+
+  if (validateEventValue(data.value, 'splitio-ga-to-split:mapper') === false)
+    return false;
+
+  const { properties } = validateEventProperties(data.properties, 'splitio-ga-to-split:mapper');
+  if (properties === false)
+    return false;
+
+  if (data.timestamp && !isNumber(data.timestamp))
+    return false;
+
+  if (data.key && validateKey(data.key, 'splitio-ga-to-split:mapper') === false)
+    return false;
+
+  if (data.trafficTypeName && validateTrafficType(data.trafficTypeName, 'splitio-ga-to-split:mapper') === false)
+    return false;
+
+  return true;
+}
+
+/**
+ * GaToSplit integration.
+ * This function provides the SplitTracker plugin to ga command queue.
+ *
+ * @param {object} sdkOptions options passed at the SDK integrations settings
+ * @param {object} storage SDK storage passed to track events
+ * @param {object} coreSettings core settings used to define an identity if no one provided as SDK or plugin options
+ */
+function GaToSplit(sdkOptions, storage, coreSettings) {
+
+  const defaultOptions = {
+    filter: defaultFilter,
+    mapper: defaultMapper,
+    // Default eventHandler stores event
+    eventHandler: function (event) {
+      storage.events.track(event);
+    },
+    // We set default identities if key and TT are present in settings.core
+    identities: (coreSettings.key && coreSettings.trafficType) ?
+      [{ key: coreSettings.key, trafficType: coreSettings.trafficType }] :
+      undefined
+  };
+
+  class SplitTracker {
+
+    // Constructor for the SplitTracker plugin.
+    constructor(tracker, pluginOptions) {
+
+      // precedence of options: SDK options (config.integrations) overwrite pluginOptions (`ga('require', 'splitTracker', pluginOptions)`)
+      const opts = Object.assign({}, defaultOptions, sdkOptions, pluginOptions);
+
+      this.tracker = tracker;
+
+      // Validate identities
+      const validIdentities = validateIdentities(opts.identities);
+
+      if (!validIdentities || validIdentities.length === 0) {
+        log.warn('No valid identities were provided. Please check that you are passing a valid list of identities or providing a traffic type at the SDK configuration.');
+        return;
+      }
```

Comment:
Where are you passing the "default identity" to this? (should only be there if identities is not defined by the user)